src/main/java/org/eclipse/emfcloud/jackson/databind/deser/RawDeserializer.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2019-2021 Guillaume Hillairet and others.
```

Comment:
Please fix the copyright header for this class by only specifying the current year and giving your own name and affiliation.

Reply:
Done.